prod_settings.py

```
@@ -0,0 +1,73 @@
+from base_settings import *
```

Ruff (0.11.9)

`from base_settings import *` used; unable to detect undefined names

(F403)